### PR TITLE
Add Russian lang

### PR DESCRIPTION
**Pull request by Rohos**
Add Russian lang

### CODE REVIEW

**Comment by Rohos** [comment]
When app routes conflict with package routes, so i add prefix on config. And change routes hardcode in routes and views

File: src/routes.php [hunk 1]
```diff
@@ -1,38 +1,78 @@
 <?php
 $middlewares = \Config::get('lfm.middlewares');
+$prefix = \Config::get('lfm.prefix');
```
Comment (youchenlee) [inline src/routes.php]: @Rohos Would you please modified it to `$prefix = \Config::get('lfm.prefix', 'laravel-filemanager');` thus we can upgrade the package seamlessly .

**Comment by youchenlee** [comment]
@Rohos Appreciate your contributions. I've made some comment to the code.
Would you please revise it for we can upgrade the package seamlessly?

File: src/routes.php (hunk 1 above)
Comment (Rohos) [inline src/routes.php]: Yes certainly already done

**Comment by youchenlee** [comment]
thank you!

**Pull request merged**